bleachbit/FileUtilities.py

```diff
@@ -80,6 +80,36 @@ def open_files_linux():
     return glob.iglob("/proc/*/fd/*")
 
 
+def get_filesystem_type(path):
+    """
+    * Get file system type from the given path
```

Comment:
Please document the return value. What are the first and second values in the tuple? 

OR: It seems it only ever uses the first value, so maybe it doesn't need to return a tuple?

Reply:
The first values is file system type and the second value is a device name like "C://". Currently we are only using the first value, but I thought driver name could be needed in the future so I added it in return value. I will add the document for return value.